docs/enhancements/bmac-reboot-hosts-on-iso-change.md

```diff
@@ -0,0 +1,131 @@
+---
+title: bmac-reboot-hosts-on-iso-change
+authors:
+  - "@mhrivnak"
+  - "@avishayt"
+creation-date: 2021-06-30
+last-updated: 2021-06-30
+---
+
+# BMAC reboots only those hosts impacted by an ISO change
+
+## Summary
```

Comment:
We should mention that this only relates to Agents that have not begun installation.  Even today any Agent that is installing/installed/error will not be rebooted.